CMakeLists.txt

```diff
@@ -292,10 +292,10 @@ endif()
 
 if(WITH_OTLP)
   set(protobuf_MODULE_COMPATIBLE ON)
-  find_package(Protobuf)
+  find_package(Protobuf PATHS ${EXTERNAL_PROTOBUF_DIR})
   if(WITH_OTLP_GRPC OR (NOT DEFINED WITH_OTLP_GRPC AND NOT DEFINED
                                                        CACHE{WITH_OTLP_GRPC}))
-    find_package(gRPC)
+    find_package(gRPC PATHS ${EXTERNAL_GRPC_DIR})
```

Comment:
Haven't seen where `${EXTERNAL_GRPC_DIR}` is defined?

Reply:
this was not defined. I' going to close the PR as the changes are not needed.